gnosis/eth/tests/ethereum_test_case.py

```diff
@@ -45,7 +45,8 @@ def setUpClass(cls):
         cls.multicall = cls.ethereum_client.multicall
         assert cls.multicall, "Multicall must be defined"
 
-    def get_ethereum_test_account(self) -> LocalAccount:
+    @classmethod
+    def get_ethereum_test_account(cls):
```

Comment:
```suggestion
    def get_ethereum_test_account(cls) -> LocalAccount:
```
I'm not sure if should be a `classmethod` because is not using anything from class.

Reply:
It's used from another `classmethod`, so it has to be a `classmethod`